src/components/Player/Player.tsx

```diff
@@ -109,6 +127,7 @@ const Player: React.FC<PlayerProps> = ({ painting, resources }) => {
         width={painting.width}
         onPlay={handlePlay}
         crossOrigin="anonymous"
+        poster={posterImage}
```

Comment:
I really like the simplicity of it but have concerns that this would conflict with visualizer/waveform.